src/util/mod.rs

```diff
@@ -1,41 +1,16 @@
-use std::{io, slice};
+#[repr(align(16))]
+pub struct WordAligned<Bytes: ?Sized>(pub Bytes);
```

Comment:
Minor nit: This should either be `align(4)` (which spirv requires) or be called something other than `WordAligned` which implies alignment to 2/4/8s (at least on x86 with a 16/32/64bit word)